jenkins/opensearch/benchmark-test.jenkinsfile

```diff
@@ -209,21 +219,29 @@ pipeline {
             agent { label AGENT_LABEL }
             steps {
                 script {
-                    if (BUNDLE_MANIFEST_URL == '') {
+                    if (BUNDLE_MANIFEST_URL == '' && DISTRIBUTION_URL == '') {
```

Comment:
Add DISTRIBUTION_VERSION too?

Reply:
Not mandatory in this context. 
We need `DISTRIBUTION_URL` or `BUNDLE_MANIFEST_URL` and the DISTRIBUTION_VERSION exception is anyway caught by the python code. 
Safe to pass for now.